src/pages/api/stageNames.ts

```diff
@@ -31,7 +31,7 @@ export const isStageNameValid = (req: NextApiRequest): InputCheck[] => {
         } else {
             error = '';
```

Comment:
This might be worth fixing as well 

Each child in a list should have a unique "key" prop.

Check the render method of `StageNames`. See https://fb.me/react-warning-keys for more information.
    in div (at stageNames.tsx:29)

On line 29 if you add the unique key that should do the trick